### ISSUE TITLE
WaitForDisk: Resource Not Found in Release 4.4.0.0

### ISSUE THREAD

**Issue by cgrycki** [PowerShell/StorageDsc#185]
#### Details of the scenario you tried and the problem that is occurring
The WaitForDisk resource is no longer found after the module is updated to release 4.4.0.0. The same configuration works in release 4.3.0.0.
#### Verbose logs showing the problem
Could not evaluate: Resource WaitForDisk could not be found.
#### Suggested solution to the issue
N/A
#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
The resource is defined below using the Puppet dsc_lite  module.
dsc {'waitfordisk_data':
    resource_name => 'WaitForDisk',
    module        => 'StorageDSC',
    properties    => {
      diskid           => $diskid,
      diskidtype       => $diskidtype,
      retryintervalsec => 60,
      retrycount       => 60,
    },
  }

#### The operating system the target node is running
OsName               : Microsoft Windows Server 2019 Standard
OsOperatingSystemSKU : StandardServerEdition
OsArchitecture       : 64-bit
WindowsVersion       : 1809
WindowsBuildLabEx    : 17763.1.amd64fre.rs5_release.180914-1434
OsLanguage           : en-US
OsMuiLanguages       : {en-US}

#### Version and build of PowerShell the target node is running

PSVersion                      5.1.17763.1                                                                             
PSEdition                      Desktop                                                                                 
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                 
BuildVersion                   10.0.17763.1                                                                            
CLRVersion                     4.0.30319.42000                                                                         
WSManStackVersion              3.0                                                                                     
PSRemotingProtocolVersion      2.3                                                                                     
SerializationVersion           1.1.0.1                                                                                 

#### Version of the DSC module that was used ('dev' if using current dev branch)
4.4.0.0

**Comment by PlagueHO** [comment]
Hi @cgrycki - I'm having trouble tracking this issue down. Can you run the following commands on your machine and report what you see?
```powershell
Install-Module -Name StorageDsc -RequiredVersion 4.4.0.0
Get-DscResource -Name WaitForDisk
```

You should see something like this:
![image](https://user-images.githubusercontent.com/7589164/50955872-17dc6500-151f-11e9-9392-9a93c1f5bd1a.png)

Is it possible to identify where Puppet is looking for the StorageDsc resource module?

**Comment by cgrycki** [comment]
I was able to determine that this is not a PowerShell/DSC issue. The new release was installed side by side with the previous release. When multiple versions of a DSC module are installed, the Puppet dsc_lite module requires an additional parameter to determine the module version to use. Unfortunately this was not reflected in the Puppet log messages and I jumped to conclusions. Thank you for your response and sorry for the trouble!
